op-guide/docker-compose.md

```diff
@@ -85,24 +85,22 @@ To customize the cluster, you can edit the `docker-compose.yml` file directly. I
 
     ```bash
     cd tidb-docker-compose
-    cp compose/values.yaml values.yaml
-    vim values.yaml
+    vi compose/values.yaml # custom the cluster size, docker image, port mapping and so on
```

Comment:
In other guides we use `vim`, for some users on some older systems they may get `vi` instead of `vim` if they type `vi`, which could lead to some confusion.